### PR TITLE
small specfile improvements

### PR DESCRIPTION
**Pull request by digitaltom**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/71539/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged fb8b971250d949ca33416b85bb59a93bc4d0ae8e into c77d6be20c592c08dd886e538dfccf7c7b023461 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72059/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged fb8b971250d949ca33416b85bb59a93bc4d0ae8e into b261ee7c2c7308f040e32a07fd6288f2f59d473c )

**Comment by lagartoflojo** [comment]
Why is all this gzip stuff no longer necessary?

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72350/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 62054d9e435a70351d21d2d4ce4ffd7639a4a41e into b261ee7c2c7308f040e32a07fd6288f2f59d473c )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72351/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged b3daca411d6de88e37372829559909266ed11eb6 into b261ee7c2c7308f040e32a07fd6288f2f59d473c )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72352/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged ef35c462126379cd3e3bed7875428a864e7cec60 into b261ee7c2c7308f040e32a07fd6288f2f59d473c )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72360/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 11734a7322ee655038496a6af220a7208bd7bc6b into b261ee7c2c7308f040e32a07fd6288f2f59d473c )

File: package/SUSEConnect.spec [hunk 1]
```diff
@@ -43,8 +47,8 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Url:            https://github.com/SUSE/connect
 
 Source:         %{mod_full_name}.gem
-Source1:        %{name}.5.gz
-Source2:        %{name}.8.gz
+Source1:        %{name}.5
+Source2:        %{name}.8
 Source3:        %{name}.example
 
 Summary:        Utility to register a system with the SUSE Customer Center
```
Comment (lagartoflojo) [inline package/SUSEConnect.spec]: https://bugzilla.suse.com/show_bug.cgi?id=964013#c12

```
Good, just one more change:
- PreReq:         update-alternatives
+ Requires(post): update-alternatives

As update-alternatives is used in post section.
```

**Comment by lagartoflojo** [comment]
Maybe you could squash all commits (except for https://github.com/SUSE/connect/pull/319/commits/ef35c462126379cd3e3bed7875428a864e7cec60) adding a link to the BNC in the commit message, since they're all part of the same thing in the end.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72383/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 378690d936751639e87ec127f3903025375b68e2 into 2f88fe30e04a26d9e78134c1e8093786bfd41921 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72399/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 8e31ed39a766405b1d19984293db4e129c585822 into 2f88fe30e04a26d9e78134c1e8093786bfd41921 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/72400/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 9a95a3aeea94bf62849be497b4a1436c3b815704 into 2f88fe30e04a26d9e78134c1e8093786bfd41921 )

**Pull request merged**